app/controllers/splash_screen_controller.rb

```diff
@@ -0,0 +1,12 @@
+class SplashScreenController < ApplicationController
+  skip_before_action :authenticate_user!
+  before_action :check_signed_in
+
+  def index; end
```

Comment:
[ OPTIONAL]
You can consider remvoving actions not used from your controllers. This makes your codebase clean and professional.